docs/data/data-grid/layout/layout.md

```diff
@@ -3,15 +3,13 @@
 <p class="description">The data grid offers multiple layout modes.</p>
 
 :::error
-By default, the data grid has **no intrinsic dimensions**. It occupies the space its parent leaves.
-The grid will raise an error in the console if its container has no intrinsic dimensions.
+By default, the data grid has **no intrinsic dimensions**.
+Instead, it takes up the space given by its parent. The grid will raise an error in the console if its container has no intrinsic dimensions.
```

Comment:
Add a line break to simplify line-by-line reviews. (It takes two line breaks for Markdown to start a new paragraph.)

```suggestion
Instead, it takes up the space given by its parent. 
The grid will raise an error in the console if its container has no intrinsic dimensions.
```

Reply:
Thanks for this! 

> It takes two line breaks for Markdown to start a new paragraph.

I see you only added one line break in the review if I'm not mistaken? Breaking to only to line 8 after ".. parent.". Your goal is to start a new paragraph?

Reply:
No, it's a convention we use: 1 sentence = 1 line as you can see on the rest of the document (the line 6 you modified was an error).

The reason is to simplify the review process. If you have one sentence per line, reviewer can add one comment/suggestion per sentence which is easier to manage.

And it also produces more meaningful git diff because git does line by line comparison so with this convention it does a sentence-per-sentence comparison.

Reply:
@alexfauquette makes perfect sense. Thanks!